package.json

```diff
@@ -65,6 +65,7 @@
 		"@dfinity/verifiable-credentials": "^0.0.4",
 		"@junobuild/analytics": "^0.0.28",
 		"@metamask/detect-provider": "^2.0.0",
+		"@solana/web3.js": "^2.0.0",
```

Comment:
⚠️ The solana web3js library has been literally compromised yesterday. From what I remember reading it on twitter late in the evening, I think it was older version, something like version v1.9.xxx but, can you please double check this version is safe?

Reply:
on it!

Reply:
Prodsec monitors this but, I thought it was better to start on a good fit. Thanks for double checking.